interface.go

```diff
@@ -68,10 +69,15 @@ func (c *Cache) GetOrCompute(key string, evaluator func() (interface{}, error))
 		mutex.Unlock()
 		return value, nil
 	}
-	if err == ErrNotFound {
-		value, err := evaluator()
-		if err == nil {
+
+	value, evaluatorErr := evaluator()
+
+	if evaluatorErr == nil {
+		// value evaluted correctly
+		if err == ErrNotFound {
+			// Key not found on cache
 			go func() {
+				// Set key to cache in gorutine
 				err := c.Set(key, value)
```

Comment:
I'm surprised linter does not complain about this, since err is never used. Could you, please, add logging of errors to Get() and Set() redis implementations? I don't think there is any other place we can log Set() errors, so lets put it there.